lab04/sections/intro.tex

```diff
@@ -1,3 +1,42 @@
 \section{Introduction}\label{sec:intro}
-Short section on the background and motivation i.e. what the experiment is about and what is being measured.
-References are appreciated \cite{lab-manual}.
+The Laplace transform, \LT, simplifies the analysis of complex networks by replacing network elements with their equivalent impedances.
```

Comment:
I might say the Laplace operator, \LT; or maybe change it to the Laplace transform, X(s) = \LT{x(t)} for a given signal x(t)... Just because \LT on it's own is just an operator.